cmd/version/version.go

```diff
@@ -37,7 +47,25 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 		Use:   "version",
 		Short: "Shows openebs kubectl plugin's version",
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("Client Version: " + rootCmd.Version)
+			k, _ := client.NewK8sClient("openebs")
```

Comment:
HardCoding openEBS namespace here, Is there any way to get the namespace.
 I am bit confused about what to pass to use this function to get the namespace.
 https://github.com/openebs/openebsctl/blob/fa583514b009899097a56584b4e33e21c49b92fd/pkg/client/k8s.go#L161
 
@vharsh

Reply:
The arguments passed(namespace) to the `NewK8sClient` aren't used in the `GetOpenEBSNamespace`, so it's getting pods from all namespaces

Reply:
Oh okay, then should I keep the current changes intact?

Reply:
> Oh okay, then should I keep the current changes intact?

No, this might be better, as `""` represents `AllNamespaces` in k8s project.
```suggestion
			k, _ := client.NewK8sClient("")
```

Reply:
Oh okay, changing it.